cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -216,8 +216,6 @@
         {% else %}
             {% include '/js/routes/' + request.path[1:] + 'index.js' ignore missing %}
         {% endif %}
-    {% else %}
```

Comment:
we're still going to need this else conditional to handle legacy pages.. unless there was a reason you removed it?